eralchemy/sqla.py

```diff
@@ -13,9 +13,20 @@
 
 def relation_to_intermediary(fk):
     """Transform an SQLAlchemy ForeignKey object to it's intermediary representation. """
+
+    # In the cases where schema name is provided, _column_tokens[1] is
+    # insufficient to correctly identify the relationship. Therefore, assuming
+    # that the schema name is _column_tokens[0], when set seems the least
+    # intrusive approach. The other one would be sending in the schema name
+    # as another parameter into this function from metadata_to_intermediary()
+    left_column = format_name(fk._column_tokens[1])
+    if fk._column_tokens[0]:
+        left_column = "{}.{}".format(format_name(fk._column_tokens[0]),
```

Comment:
I require a test of this fix preferably in in integration with a real postgresql

Reply:
Amended existing schema test to assert that column names match full table names.
Hope that is enough.

Reply:
I need automated testing for this feature possibly in integration with a real Postgres.

Reply:
The current test uses the local instance of a postgres database (named `test` and created [inside `.travis.yml`](https://github.com/Alexis-benoist/eralchemy/blob/master/.travis.yml#L14-L15). Is that not enough?